### DISCUSSION TITLE
请问能支持排序吗？

### DISCUSSION THREAD

**Discussion by yanchao7511461** [spencerwooo/onedrive-vercel-index#165] [Ideas]
可以支持修改时间、大小、文件(夹)名称的表头排序吗?  现在文件很多.. 找起来不方便，如果有排序会方便很多

**Comment by spencerwooo** [comment]
某一天如果心情好了可能会写这个功能吧 🥰

**Reply by superen7527** [reply]
你好 请问现在支持了吗 不太会更新合并fork 感觉有点麻烦 但是想要排序这个功能 现在用的旧版 如果有排序我就更新一下下 打扰了 不知道哪里看更新日志..
